src/app/portlets/content-types/content-types-routing.module.ts

```diff
@@ -1,7 +1,6 @@
-import { ContentTypesEditComponent } from './edit';
```

Comment:
<img width="455" alt="Screen Shot 2019-09-04 at 10 38 01 AM" src="https://user-images.githubusercontent.com/751424/64274225-289f7800-cf00-11e9-8c4c-cf5f68860e11.png">

All this should be in shared.